aws/resource_aws_eks_cluster.go

```diff
@@ -145,6 +145,14 @@ func resourceAwsEksCluster() *schema.Resource {
 							MinItems: 1,
 							Elem:     &schema.Schema{Type: schema.TypeString},
 						},
+						"public_access_cidrs": {
+							Type:     schema.TypeSet,
+							Optional: true,
```

Comment:
It appears that EKS provides a default `0.0.0.0/0` entry when this argument is not configured:

```
--- FAIL: TestAccAWSEksCluster_basic (1307.89s)
    testing.go:640: Step 0 error: After applying this step, the plan was not empty:
        
        DIFF:
        
        UPDATE: aws_eks_cluster.test
... omitted for clarity ...
          vpc_config.0.public_access_cidrs.#:     "1" => "0"
          vpc_config.0.public_access_cidrs.0:     "0.0.0.0/0" => ""
... omitted for clarity ...
```

Given that the Terraform Plugin SDK currently does not support a way to configure a [`Default` for `TypeList`/`TypeSet`](https://github.com/hashicorp/terraform-plugin-sdk/issues/142), our only recourse here is to add `Computed: true` to this for now.

```suggestion
							Optional: true,
							Computed: true,
```